sender_integration_test.go

```diff
@@ -250,7 +252,7 @@ func TestE2EValidWrites(t *testing.T) {
 					Symbol("sym_col", "test_ilp2").
 					Float64Column("double_col", 11.2).
 					Int64Column("long_col", 11).
-					Long256Column("long256_col", "0x123a3i").
+					Long256Column("long256_col",  big.NewInt(7423093023234231)).
```

Comment:
Please use `SetString` here and above. This particular `bit.Int` should also be greater than 64 bits, so that we make sure that the endianess is the same in the input and the output.